azkaban-common/src/main/java/azkaban/project/FeatureFlag.java

```diff
@@ -0,0 +1,35 @@
+package azkaban.project;
+
+/**
+ * A Feature Flag class contains enum of all feature flags.
+ * */
+public enum FeatureFlag {
```

Comment:
The `FeatureFlag` here is only for `project`, right? 
Is there anywhere else can generalize this? like in azkaban-common?

Reply:
I was intended to use featureFlag for projects, but feel free to extend it. The initial thought is flow has props concept but we don't have project level props, and I know some other users has mentioned this project level props to us. So I thought it is worthwhile to have a standalone class for that.